specification/resources/1-clicks/examples/curl/1_click.yml

```
@@ -0,0 +1,3 @@
+lang: cURL
+source: |-
+  curl -X GET -H "Content-Type: application/json"  -H "Authorization: Bearer b7d03a6947b217efb6f3ec3bd3504582" "https://api.digitalocean.com/v2/1-clicks" 
```

This is non-blocking and could be done in a subsequent change.
It might be helpful to split the curl commands into multiple lines.

```suggestion
  curl -X GET \
  -H "Content-Type: application/json"  \
  -H "Authorization: Bearer b7d03a6947b217efb6f3ec3bd3504582" \
  "https://api.digitalocean.com/v2/1-clicks" 
```

---

I had the same thought. Think we can probably come up with some scripting to do this.

---

Took a stab at it. This seems to do what we want:

```
grep -rl '\-H "Content-Type:' specification/*/*/examples/curl | xargs sed -i 's/-H "Content/\\\n    -H "Content/g'
grep -rl '\-H "Authorization:' specification/*/*/examples/curl | xargs sed -i 's/-H "Authorization/\\\n    -H "Authorization/g'
grep -rl "\-d '{" specification/*/*/examples/curl | xargs sed -i 's/-d/\\\n    -d/g'
grep -rl '"https://api' specification/*/*/examples/curl | xargs sed -i 's|"https://api|\\\n    "https://api|g'
```